.github/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - master
+
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  rust:
+    uses: primait/shared-github-actions/.github/workflows/rust-ci.yml@rust-ci-v1
+    with:
+      enforce-coverage-threshold: false
+      run-coverage-task: false
+
+      rust-toolchain-version: "1.72.0"
+
+  alls-green:
+    if: always()
+    needs:
+      - rust
+    runs-on: [self-hosted, k8s-small]
+    container:
+      image: public.ecr.aws/prima/python:3.9.10-3
```

Comment:
related to the previous comment, public github runners have python installed, so alls-green should work without this